src/PageContent/Settings/Settings.js

```diff
@@ -111,7 +110,7 @@ class Settings extends Component {
                                 <div>Loading...</div>  
                             }
                         </div>
-
+                        <div> {this.props.availabilities}</div>
```

Comment:
I can't quite figure out what this line of code is doing, it looks like Settings never has any availabilities props? It has this.props.account.availabilities but it's not JSX so it wouldn't be rendered. Let me know if I'm missing something